pkg/odo/cli/component/list.go

```diff
@@ -310,7 +304,7 @@ func (lo *ListOptions) Run() (err error) {
 			return
 		}
 	} else {
-		combinedComponents := component.GetMachineReadableFormatForCombinedCompList(components, devfileComponents)
+		combinedComponents := component.GetMachineReadableFormatForCombinedCompList(components, devfileComponents.Items)
```

Comment:
Variable names could be different to indicate s2i components and devfile components. Also, in one case we need the `.Items` whereas not in the other. Can we have uniformity?

Reply:
variable names are different to indicate s2i component and devfile components already, one is called `components` and other is `devfileComponents`. Am I missing something? also 

> Also, in one case we need the `.Items` whereas not in the other. Can we have uniformity?

How can I have uniformity? use the Items for everywhere?

Reply:
resolved the `Also..` portion

Reply:
> variable names are different to indicate s2i component and devfile components already, one is called components and other is devfileComponents. Am I missing something?

My bad. Should have made it clear that I was requesting something like `s/components/s2iComponents` for the variable to store s2i components.

Reply:
resolved